ra/runtime/src/main/java/io/quarkus/artemis/jms/ra/ArtemisResourceAdapterFactory.java

```diff
@@ -86,6 +97,14 @@ public MessageEndpoint wrap(MessageEndpoint endpoint, Object resourceEndpoint) {
         return new JMSMessageEndpoint(endpoint, (MessageListener) resourceEndpoint);
     }
 
+    private static int count(String text, String find) {
+        int count = 0, length = find.length();
+        for (int index = 0; (index = text.indexOf(find, index)) != -1; index += length) {
+            count++;
+        }
+        return count;
```

Comment:
I do not like the two variable declarations in one line.

Also, we could reduce this to an onleliner (and possibly inline the method):
```suggestion
        return text.split(find).length - 1;
```

 the `-1` in the `split(...)` call signals that trailing matches should not be discarded. This would be relevant if the `text` ends with `find`.

Reply:
`split` is a regex, so you need to escape.

Reply:
w.l.o.g.: yes
If we inline: no

Reply:
I split the declaration into 2 lines, I'd prefer to keep the implementation as is, thanks